src/sync/incomingSyncUtils.js

```diff
@@ -551,25 +563,33 @@ export function sanityCheckIncomingRecord(recordType, record) {
       canBeBlank: ['category', 'code', 'type'],
     },
     TransactionBatch: {
-      cannotBeBlank: ['item_ID', 'item_line_ID', 'expiry_date', 'quantity', 'cost_price',
-                      'sell_price'],
+      cannotBeBlank: [
+        'item_ID',
+        'item_line_ID',
+        'expiry_date',
+        'quantity',
+        'cost_price',
+        'sell_price',
+      ],
       canBeBlank: ['item_name', 'batch', 'expiry_date', 'pack_size', 'cost_price', 'sell_price'],
     },
   };
   if (!requiredFields[recordType]) return false; // Unsupported record type
   const hasAllNonBlankFields = requiredFields[recordType].cannotBeBlank.reduce(
     (containsAllFieldsSoFar, fieldName) =>
       containsAllFieldsSoFar &&
-      record[fieldName] !== null &&  // Key must exist
-      record[fieldName].length > 0,  // And must not be blank
-      true);
+      record[fieldName] !== null && // Key must exist
+      record[fieldName].length > 0, // And must not be blank
+    true
+  );
   if (!hasAllNonBlankFields) return false; // Return early if record already not valid
   const hasRequiredFields = requiredFields[recordType].canBeBlank.reduce(
     (containsAllFieldsSoFar, fieldName) =>
       containsAllFieldsSoFar &&
-      record[fieldName] !== null &&    // Key must exist
+      record[fieldName] !== null && // Key must exist
       record[fieldName] !== undefined, // May be blank, i.e. just ''
-      hasAllNonBlankFields); // Start containsAllFieldsSoFar as result from hasAllNonBlankFields
+    hasAllNonBlankFields
+  ); // Start containsAllFieldsSoFar as result from hasAllNonBlankFields
```

Comment:
Nothing changed, I originally tried something here and didn't revert the pretty-ness.